### ISSUE TITLE
Compiler API is not reporting all the errors .

### ISSUE THREAD

**Issue by ashishpawar517** [ashishpawar517/AutoCompiler#14]
Consider the following code 
```java
// Dummy file for testing purpose
public class Demo{

      public static void main(String[] args) 
      {
           System.out.println("hello world")
           int array = new int[10];
      }
}

```
Compiler API first reports only semicolon in first try. After than it will handle the array [] missing .

**Comment by pawarashish564** [comment]
like javac , java compiler API first reports all first level errors then it reports all other errors . @tauseefansari @Afif79
